src/RenderJson.js

```diff
@@ -57,4 +56,24 @@ module.exports = class RenderJson {
     }
     return output
   }
+
+  formatLibraries() {
+    if (!this.data.lhr) return []
+
+    const jsLibrariesResult = this.data.lhr.audits['js-libraries']
+    if (
+      jsLibrariesResult &&
+      jsLibrariesResult.details &&
+      jsLibrariesResult.details.items &&
+      jsLibrariesResult.details.items.length > 0
+    ) {
+      return jsLibrariesResult.details.items.map(jsLib => {
+        return {
+          name: jsLib.name,
+          version: jsLib.version || '(version not avaliable)'
```

Comment:
```suggestion
          version: jsLib.version || '(version not available)'
```

Reply:
as pointed out at https://github.com/lirantal/is-website-vulnerable/pull/31#issuecomment-546688517 let's resolve this first before merging @mixth

Reply:
Done :) Thanks @rajkumaar23! I do fix them both in `json` and `console`.

Reply:
awesome work you two 👏